views/transaccion_completa_mall_deferred/installments.pug

```diff
@@ -35,7 +35,7 @@ block step
       p Para confirmar una transacción debemos pasar los siguentes datos.
     .col.m8 
       .card-panel
-        form(action="/transaccion_completa_mall_deferred/commit", method="POST")
```

Comment:
Same here, is correct change to _transaccion_completa_mall_, if this is _transaccion_completa_mall_deferred_ view?